assignment3.txt

```diff
@@ -0,0 +1,14 @@
+Can Parlar
+
+1.A. O(n)
+
+1.B. O(n^3)
```

Comment:
Not quite! you should have given an explanation so I can tell you where you went wrong. the time complexity of this is n^2 + n, which evaluates to O(n^2). The n^2 comes from the nested for loops in the second half of the code snippet and the n comes from the first for loop.